chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -17,16 +19,16 @@
 end
 
 directory "/var/run/ceph" do
-  owner "root"
-  group "root"
-  mode "0755"
+  owner "ceph"
+  group "ceph"
+  mode "0770"
   action :create
 end
 
 directory "/var/log/ceph" do
-  owner "root"
-  group "root"
-  mode "0755"
+  owner "ceph"
+  group "ceph"
+  mode "3770"
```

Comment:
I don't know why that's necessary offhand, but it matches what's in the ceph spec file:

`%attr(3770,ceph,ceph) %dir %{_localstatedir}/log/ceph/`